src/main.py

```diff
@@ -36,12 +36,12 @@ def start():
     if set_token is False:
         pause("No token found! Please login to Drillster on a browser and keep the tab open! Press any key to exit...")
 
-    start_time = time.time()
```

Comment:
Timer started before selecting drills, moved it over to only start when drills are selected